src/screens/ContactScreen/index.js

```diff
@@ -95,7 +104,11 @@ const ContactScreen = ({navigation}) => {
           })) || [];
 
         const dataUser = [...followedUsers, ...unfollowedUsers];
-        setUsers(dataUser);
+        setUsers(
+          dataUser?.filter((item) =>
+            isAddParticipant ? !existParticipants.includes(item.username) : item
+          )
+        );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the setting of users now includes a filter based on whether the current operation is adding participants and whether the user already exists in the participant list. Again, ensure that the edge case of an empty array is handled appropriately in the UI.
```diff
-    setUsers(dataUser);
+    setUsers(
+      dataUser?.filter((item) =>
+        isAddParticipant ? !existParticipants.includes(item.username) : item
+      )
+    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->